samcli/commands/local/cli_common/options.py

```diff
@@ -173,16 +173,16 @@ def invoke_common_options(f):
                 "--docker-volume-basedir",
                 "-v",
                 envvar="SAM_DOCKER_VOLUME_BASEDIR",
-                help="Specifies the location basedir where the SAM file exists. If the Docker is running on "
-                "a remote machine, you must mount the path where the SAM file exists on the docker machine "
-                "and modify this value to match the remote machine.",
+                help="Specify the location basedir where the SAM template exists. If Docker is running on "
```

Comment:
Thinking out loud question: should we keep specifying "Docker" in our help text/docs or should we move away from that and start using terminology like "Container engine"?

Reply:
Good call, I'd keep that to separate PR to address those concerns.